src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -604,44 +604,118 @@ internal BoundStatement BindLocalDeclarationStatement(LocalDeclarationStatementS
 
         private BoundStatement BindDeclarationStatementParts(LocalDeclarationStatementSyntax node, DiagnosticBag diagnostics)
         {
-            var typeSyntax = node.Declaration.Type.SkipRef(out _);
-            bool isConst = node.IsConst;
+            var variableList = node.Declaration.Variables;
```

Comment:
These look like they can stay in the `else` below, since they're not used in the `using var` form.